config.tpl.yml

```diff
@@ -41,6 +41,14 @@ APPS:
           weekends:
             - 08:00-23:00
 
+  - name: notify-api-sms-callbacks
+    min_instances: {{ MAX_INSTANCE_COUNT_MEDIUM }}
```

Comment:
hey Becca, why is the env var for `min_instances` called `MAX_INSTANCE_COUNT_MEDIUM`? It's a bit confusing, but probably I'm missing something.

Reply:
Ahh, I see, it's one of the instance limits, still, the name feels weird.

Reply:
Yes - these variable names make the file more confusing than they should be. I can create a new variable but it doesn't help that I need to go to another file to read the value of the variable.

Reply:
that is true, what is the advantage of us having the values in different file? Is it because we are using them in several places and want to have single place of truth?